### PR TITLE
Update actions/cache in workflow

### PR DESCRIPTION
**Pull request by haslinghuis**
Reference: https://github.com/marketplace/actions/cache

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

**Comment by nerdCopter** [comment]
error makes no sense: 

> Unable to resolve action `actions/cache@v4`, unable to find version `v4`

**Comment by haslinghuis** [comment]
Do not merge until rerun shows v4 is actually available

**Comment by nerdCopter** [comment]
it works after #687 is merged in. (see testing pr on personal repo: https://github.com/nerdCopter/blackbox-log-viewer/pull/1)

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
